packages/core/src/pages/index.tsx

```diff
@@ -27,6 +28,7 @@ function Page({
   globalSections,
   serverData,
 }: Props) {
+  useGeolocation()
```

Comment:
question: should we also add this in the search page?

Reply:
It got me thinking about it too, but we should agree that is not common at all to a user come directly from a search page's link. I'm divided, but if we agree to add to the search page, maybe I'll try to call this hook on `_app.tsx` for all the pages, what do you think?